### ISSUE TITLE
Add support of TEXT_MAP format for injection/extraction of the span data

### ISSUE THREAD

**Issue by elgris** [instana/python-sensor#13]
## User story

As an application developer who collects tracing data in a very tricky way
I want Instana tracing library to support `TEXT_MAP` format for injection/extraction of the span data
So that I don't need to implement this standard way of injection/extraction myself

## Explanation

Currently instana tracer is able to inject/extract span data using `HTTPPropagator` which works fine for tracing HTTP requests. However, in my case it's needed to work with data coming from a queue. And queue messages don't support HTTP headers :)

Another reason for adding support of `TEXT_MAP` format is that it's suggested by Opentracing standard: http://opentracing.io/documentation/pages/api/cross-process-tracing.html#required-injectextract-carrier-formats

## Acceptance criteria

- [ ] It is possible to extract span data with a following call:
```
tracer.extract(
                    format=Format.TEXT_MAP,
                    carrier=trace,
                )
```
- [ ] The same goes for `tracer.inject` call

**Comment by elgris** [comment]
Just in case, here's an implementation from basictracer: https://github.com/opentracing/basictracer-python/blob/master/basictracer/text_propagator.py

Perhaps, it's possible to reuse?

**Comment by pglombardo** [comment]
This was fixed in #16.  Thanks @elgris.  Closing.
